source/appModules/explorer.py

```diff
@@ -195,6 +195,21 @@ def _get_TextInfo(cls):
 		return cls.TextInfo
 
 
+class WorkerW(IAccessible):

```

Comment:
Please add a doc string that indicates that this is Windows 7 specific code. Makes it easier to remove it if/when we drop support for Windows 7.

Reply:
The class cannot be removed entirely even when we would no longer support Windows 7, because it also handles #6671. The commends inside event_gainFocus explains which part is Windows 7 specific.